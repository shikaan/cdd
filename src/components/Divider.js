import styled from 'styled-components'

export default styled.hr(({ theme }) => `
  height: 1px;
  color: ${theme.color.lightGrey};
`)
