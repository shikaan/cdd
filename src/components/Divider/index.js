import styled from 'styled-components'
import { Size } from '~theme'

export default styled.hr`
  color: ${({ theme }) => theme.color.lightGrey};
  height: 1px;
  margin: ${new Size(2)}
`
