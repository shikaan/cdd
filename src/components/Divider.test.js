import React from 'react'
import {shallow} from 'enzyme';
import 'jest-styled-components';

import Divider from "./Divider.js";

describe('Divider', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Divider/>)

    expect(wrapper).toMatchSnapshot()
  })
})
