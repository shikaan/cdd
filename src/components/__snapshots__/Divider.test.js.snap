// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Divider renders correctly 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          [Function],
        ],
      },
      "displayName": "styled.hr",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "hr",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
